services/hanke-service/build.gradle.kts

```diff
@@ -165,3 +180,17 @@ tasks.register("installGitHook", Copy::class) {
 tasks.named("build") { dependsOn(tasks.named("installGitHook")) }
 
 tasks.named("check") { dependsOn(tasks.named("integrationTest")) }
+
+tasks.named("processResources") { dependsOn(tasks.named("copyEmailTemplates")) }
+
+mjml {
```

Comment:
The plugin thing is a very good improvement.

However, should this part have some comments on what these mean? 

For example, I have no idea what `juiceOptions.set("{\"preserveImportant\": true}")` does.

Reply:
I added a link to MJML documentation where the config options are explained. Also, a separate link to html-minifier documentation for the minifyOptions.

The juice-stuff didn't do anything for us, so I removed it. It was copied along an example config.